### PR TITLE
Update issue template

### PR DESCRIPTION
**Pull request by Latropos**
## Summary

This is just a proposal

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 1]
```diff
@@ -145,7 +167,7 @@ body:
       description: What device you are experiencing this problem on? Specify full device name along with the version of the operating system it's running.
       placeholder: ex. Samsung Galaxy A22 (Android 12)
 
-  - type: dropdown
+  - type: checkboxes
```
Comment (tomekzaw) [inline .github/ISSUE_TEMPLATE/bug-report.yml:170]: There's a particular reason why we chose to use dropdown here &ndash; each checkbox inside issue description is treated like a task and shown like this in the issues list:

Zrzut ekranu 2023-08-1 o 00 00 40

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 2]
```diff
@@ -20,23 +20,44 @@ body:
...
   - type: textarea
     id: description
     attributes:
       label: Description
-      description: Please provide a clear, concise and descriptive explanation of what the bug is. Include screenshots or a video if needed. Tell us what were you expecting to happen instead of what is happening now.
+      description: | 
+        Please provide a clear, concise and descriptive explanation of what the bug is. 
+        Make sure you tell us:
+        - [ ] What is the **actual** behavior
+        - [ ] What is the **expected** behavior
```
Comment (tomekzaw) [inline .github/ISSUE_TEMPLATE/bug-report.yml:32]: Have you checked how does it look like in the final rendered issue template? (You can check it on some private repository).

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 3]
```diff
@@ -20,23 +20,44 @@ body:
...
+      value: |
+        ### What happens? 
+
+        ___
+        <details>
+        ```
+        If you have long snippets of code or logs, place them HERE
+        ```
+        </details>
```
Comment (tomekzaw) [inline .github/ISSUE_TEMPLATE/bug-report.yml:41]: Not a huge fan of using UPPERCASE letters for whole words.

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 4]
```diff
@@ -20,23 +20,44 @@ body:
...
     validations:
       required: true
+    id: screenshot
+    attributes:
+      label: Screenshot/Recording
+      description: If possible, please add screenshots or recording. 🙏
```
Comment (tomekzaw) [inline .github/ISSUE_TEMPLATE/bug-report.yml:47]: We don't use emojis anywhere, why should we use it here?

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 5]
```diff
@@ -20,23 +20,44 @@ body:
...
 
   - type: textarea
     id: steps-to-reproduce
     attributes:
       label: Steps to reproduce
-      description: Provide a detailed list of steps that reproduce the issue.
+      description: |
+       How to reproduce your issue? 
+       > **Warn**
```
Comment (tomekzaw) [inline .github/ISSUE_TEMPLATE/bug-report.yml:55]: Again, some custom formatting that won't work.

**Review by tomekzaw — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/bug-report.yml (hunk 1 above)
Comment (Latropos) [reply .github/ISSUE_TEMPLATE/bug-report.yml:170]: Yes, but github is going to fix this soon: https://github.com/orgs/community/discussions/5197#discussioncomment-5966864

File: .github/ISSUE_TEMPLATE/bug-report.yml (hunk 4 above)
Comment (Latropos) [reply .github/ISSUE_TEMPLATE/bug-report.yml:47]: To get some attention. I think we should try to force people to read the description. Sometimes we get bug descriptions, that consist of error log without any explanation

Comment (tjzel) [reply .github/ISSUE_TEMPLATE/bug-report.yml:47]: I like the emoji but it should be without the trailing dot on the sentence

**Pull request closed without merging**